package.json

```diff
@@ -6,18 +6,23 @@
   "author": "Shahed Nasser",
   "license": "ISC",
   "scripts": {
+    "start": "npm run watch",
     "compile": "lessc src/sbuttons.less dist/sbuttons.css && npm run prettier && npm run clean-css",
-    "watch": "concurrently \"less-watch-compiler src dist\" \"onchange -v dist/sbuttons.css -- npm run clean-css \" ",
     "prettier": "prettier --write \"src/**/*.less\" \"dist/sbuttons.css\" \"assets/**\" ",
+    "lite": "lite-server",
     "check-prettier": "prettier --check \"src/**/*.less\" \"dist/sbuttons.css\" ",
-    "clean-css": "cleancss -o dist/sbuttons.min.css dist/sbuttons.css"
+    "clean-css": "cleancss -o dist/sbuttons.min.css dist/sbuttons.css",
+    "watch:less": "onchange -v src/*.less -- npm run compile",
+    "watch": "concurrently 'npm run watch:less' 'npm run lite'"
```

Comment:
It seems concurrently required double quotations instead of single quotations so please replace the command with `concurrently \"npm run watch:less\" \"npm run lite\"`

Reply:
I see. Ok